R/dev-help.R

```diff
@@ -57,6 +57,13 @@ print.dev_topic <- function(x, ...) {
   cli::cli_inform(c("i" = "Rendering development documentation for {.val {x$topic}}"))
 
   type <- arg_match0(x$type %||% "text", c("text", "html"))
+
+  # use rstudio's previewRd() is possible
+  if (type == "html" && is_rstudio() && is_installed("rstudioapi") && rstudioapi::hasFun("previewRd")) {
```

Comment:
When https://github.com/rstudio/rstudio/pull/12111 will be merged, we can use the build number as `rstudioapi::hasFun(version_needed = )`, currently is: 

```
> rstudioapi:::getVersion()
[1] ‘2022.12.0.250’
```

Reply:
Actually, we would only need this new version if there are macros right ? So here we could decide to use `previewRd()` if : the version is recent enough or there are no macros.